src/platform-includes/performance/automatic-performance-metrics/android.mdx

```diff
@@ -2,3 +2,25 @@ If configured, the Android SDK automatically collects the following performance
 
 * <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#app-start-instrumentation">Cold and warm app start time</PlatformLink>.
 * <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#slow-and-frozen-frames">Slow and frozen frame rendering</PlatformLink>.
+
+## Application Not Responding (ANR) Rate
+
+<Include name="early-adopter-note.mdx" />
+
+If configured, the Android SDK reports [Application Not Responding (ANR)](/platforms/android/configuration/app-not-respond/) errors if the main UI thread of the application is blocked for more than five seconds. ANR rate is the percentage of distinct users who have experienced any kind of ANR during the specified time period. Foreground ANR rate is the percentage of distinct users who have experienced an ANR when the app was running in the foreground. It is more likely that foreground ANRs were perceieved by the user.
+
```

Comment:
```suggestion
Application Not Responding (ANR) errors are triggered when the main UI thread of an application is blocked for more than five seconds. When configured, Sentry's Android SDK reports [ANR errors](/platforms/android/configuration/app-not-respond/). 

```

Reply:
What do you think of this? I think for the user experience it makes sense to first define what we're talking about and then let them know that we support it. I also think that since we've devoted sections to User-perceived ANR rate and Foreground ANR, we don't need to introduce it in the first paragraph.